### DISCUSSION TITLE
FSF decided to stand up for RMS.

### DISCUSSION THREAD

**Discussion by dodobyte** [rms-support-letter/rms-support-letter.github.io#4856] [General]
I think we should all donate even a small amount.
https://twitter.com/fsf/status/1376714994721689601
https://my.fsf.org/donate

**Comment by greenhorse-effect** [comment]
Twitter needs nonfree JS, here's alternative URL: https://nitter.snopyta.org/fsf/status/1376714994721689601

**Reply by AnimusPEXUS** [reply]
imho, RetroShare have to be fixed - currently it's the most advanced free software communication software.

**Comment by Aerocatia** [comment]
Gee those twitter replies are disgusting. How can people be so horribly misinformed and continue to spread an easily debunked lie in regards to what Stallman said in 2019?

**Comment by pda0** [comment]
> I think we should all donate even a small amount

Meet the new monthly associate member.
